doc/install.rst

```diff
@@ -79,7 +75,7 @@ First, we must configure conda to get packages from the
 Now we can create a new conda environment with Python and all our dependencies installed
 (we'll call it ``pygmt`` but you can change it to whatever you want)::
 
-     conda create --name pygmt python=3.6 pip numpy pandas xarray packaging gmt=6.0.0
+     conda create --name pygmt python=3.8 pip numpy pandas xarray packaging gmt=6.0.0
```

Comment:
```suggestion
     conda create --name pygmt python=3.8 pip numpy pandas xarray packaging gmt
```

Reply:
I'm fine with this. You'll get a merge conflict with the netcdf branch though. Do you want to just merge the netcdf one into this PR?

Reply:
Yes, merged that branch in to make life easier.